README.md

```diff
@@ -88,6 +88,53 @@ git clone git@github.com:CMSgov/beneficiary-fhir-data.git ~/workspaces/bfd/benef
     ```
 1. Install pre-commit hooks `mvn -f apps initialize`
 
+### Adding Reference to AWS Code Artifactory
```

Comment:
as discussed, lets move this down below native setup and modify native setup to include a new step 1 of running 'mvn clean install' in the bfd-fda-data project (which should populate the artifact in the m2 repo and be easier way for new engineers to get started)

Reply:
done